sunpy/map/tests/test_mapbase.py

```diff
@@ -329,58 +329,57 @@ def calc_new_matrix(angle):
     return np.matrix([[c, -s], [s, c]])
 
 
-def test_rotate():
-    aia_map = sunpy.map.Map(sunpy.AIA_171_IMAGE)
-    rotated_map_1 = aia_map.rotate(20)
+def test_rotate(aia171_test_map):
+    rotated_map_1 = aia171_test_map.rotate(20)
     rotated_map_2 = rotated_map_1.rotate(20)
-    assert rotated_map_2.center == rotated_map_1.center == aia_map.center
+    assert rotated_map_2.center == rotated_map_1.center == aia171_test_map.center
     np.testing.assert_allclose(rotated_map_1.rotation_matrix,
-                               np.dot(aia_map.rotation_matrix,
+                               np.dot(aia171_test_map.rotation_matrix,
                                       calc_new_matrix(20).T))
     np.testing.assert_allclose(rotated_map_2.rotation_matrix,
-                               np.dot(aia_map.rotation_matrix,
+                               np.dot(aia171_test_map.rotation_matrix,
                                       calc_new_matrix(40).T))
 
     # Rotation of a map by a non-integral multiple of 90 degrees expands the map
     # and assigns the value of 0 to corner pixels. This results in a reduction
     # of the mean for a map of all non-negative values.
-    assert rotated_map_2.shape > rotated_map_1.shape > aia_map.shape
+    assert rotated_map_2.shape > rotated_map_1.shape > aia171_test_map.shape
     np.testing.assert_allclose(rotated_map_1.data[0, 0], 0., atol=1e-7)
     np.testing.assert_allclose(rotated_map_2.data[0, 0], 0., atol=1e-7)
-    assert rotated_map_2.mean() < rotated_map_1.mean() < aia_map.mean()
+    assert rotated_map_2.mean() < rotated_map_1.mean() < aia171_test_map.mean()
 
-    rotated_map_3 = aia_map.rotate(0, scale=1.5)
-    assert rotated_map_3.mean() > aia_map.mean()
+    rotated_map_3 = aia171_test_map.rotate(0, scale=1.5)
+    assert rotated_map_3.mean() > aia171_test_map.mean()
 
     # Mean and std should be equal when angle of rotation is integral multiple
     # of 90 degrees for a square map
-    rotated_map_4 = aia_map.rotate(90, scale=1.5)
+    rotated_map_4 = aia171_test_map.rotate(90, scale=1.5)
     np.testing.assert_allclose(rotated_map_3.mean(), rotated_map_4.mean(), rtol=1e-3)
     np.testing.assert_allclose(rotated_map_3.std(), rotated_map_4.std(), rtol=1e-3)
-    rotated_map_5 = aia_map.rotate(180, scale=1.5)
+    rotated_map_5 = aia171_test_map.rotate(180, scale=1.5)
     np.testing.assert_allclose(rotated_map_3.mean(), rotated_map_5.mean(), rtol=1e-3)
     np.testing.assert_allclose(rotated_map_3.std(), rotated_map_5.std(), rtol=1e-3)
```

Comment:
You will have to widen the tolerance for this test (e.g., set `rtol` to 2e-3) to get it to pass on the new test map.  I have not looked into why.